### PR TITLE
HW-264: Sync with Master - Styling structure improvements 

### PR DESCRIPTION
**Pull request by meladawy**
1-  Pull and fix conflict from civicrm/master branch 
2- Remove shadows from panel class and use ".panel-default" class in markup instead 
3- Remove select2 overriding styles and move it to Mosaico extension
4- Move mosaico related variables to mosaico extension and remove unwanted variables

### CODE REVIEW

File: scss/bootstrap/overrides/style/_forms.scss [hunk 1]
```diff
@@ -12,10 +12,10 @@ label {
 }
 
 .form-control {
-  @include box-shadow(inset 0 0 3px 0 rgba(0, 0, 0, 0.2));
+  @include box-shadow($crm-form-control-shadow);
 
   &:focus {
-    @include box-shadow(inset 0 0 3px 0 rgba(0, 0, 0, 0.2));
+    @include box-shadow($crm-form-control-shadow);
```
Comment (boriszaydlin) [inline scss/bootstrap/overrides/style/_forms.scss]: remove this empty line

File: scss/bootstrap/vendor/select2/select2-bootstrap.scss [hunk 2]
```diff
@@ -1,250 +1,94 @@
...
-  .select2-choice .select2-arrow {
+.form-control .select2-choice .select2-arrow {
+    border-radius: 0 2px 2px 0;
     background: transparent;
     border-left: 0 none;
-    border-radius: 0 2px 2px 0;
     right: 4px;
     top: 2px;
-  }
-
```
Comment (boriszaydlin) [inline scss/bootstrap/vendor/select2/select2-bootstrap.scss:22]: Are you sure about deleting all of this?

File: scss/bootstrap/vendor/select2/select2-bootstrap.scss [hunk 3]
```diff
@@ -1,250 +1,94 @@
...
-.form-group .select2-container {
-  background: $crm-white;
-  border: 1px solid $crm-darker-grayblue;
-  box-shadow: $crm-form-control-shadow;
-  height: 30px;
-  padding: 0;
-
-  .select2-choice {
+.form-control .select2-choice {
     border: 0;
```
Comment (boriszaydlin) [inline scss/bootstrap/vendor/select2/select2-bootstrap.scss:2]: Can you please globally change tab width to 2 spaces?

Comment (meladawy) [reply scss/bootstrap/vendor/select2/select2-bootstrap.scss:2]: Mmmm do u mean for this file only? I just reverted this file to be like master branch.
git checkout origin/master select2-bootstrap.scss
no need to modify this file any more. all the custom styling has been moved to another extension. as per @AkA84  recommendation, we shouln't touch this file.

File: scss/bootstrap/vendor/select2/select2-bootstrap.scss (hunk 2 above)
Comment (meladawy) [reply scss/bootstrap/vendor/select2/select2-bootstrap.scss:22]: Yes

File: scss/bootstrap/vendor/select2/select2-bootstrap.scss (hunk 3 above)
Comment (boriszaydlin) [reply scss/bootstrap/vendor/select2/select2-bootstrap.scss:2]: Okay, got it

**Review by boriszaydlin — APPROVED** [review]

**Pull request merged**